src/components/calcite-modal/calcite-modal.scss

```diff
@@ -148,6 +146,8 @@ slot[name="header"]::slotted(*) {
   padding: $baseline;
   height: 100%;
   overflow: auto;
+  max-height: calc(100vh - 14rem);
```

Comment:
`calc` is high-key my favorite new CSS feature of the past several years. More than flexbox or grid.